backend/svirt.pm

```diff
@@ -21,13 +21,19 @@ use testapi qw(get_var get_required_var check_var);
 
 use IO::Select;
 
+use Class::Accessor "antlers";
+has instance => (is => "rw", isa => "Num");
+has name     => (is => "rw", isa => "Str");
+
```

Comment:
Sure. But (1) I have no idea how do I, while in backend, get those properties from console, and (2) I am not sure if it's available at that point as console gets initialized later, I believe.